source/user/lo.rst

```diff
@@ -0,0 +1,66 @@
+.. _user_lo:
+
+**********************
+Localized Orbitals
+**********************
+
+*Modules*: :mod:`lo`
+
+Introduction
+============
+A molecular orbital is usually delocalized, i.e. its spatial distribution is
+all over the whole molecular rather than localized around several atoms or bonds.
+However, one can choose a proper unitary rotation :math:`U`
+
+.. math::
+
+    \phi = \psi U
+
+such that the resulting
+orbitals :math:`\phi` are as local as possible. This is typically achieved by two kinds of
+methods: One is by projection onto a pre-defined local orbital set
+(e.g., AO, pseudopotential orbitals); The other is by numerical optimization on
+a cost function :math:`f`, which can be chosen as different localization
+criterion, e.g., in Boys localization, the dipole (spread) of a molecule,
```

Comment:
```suggestion
methods. 
The first is to project the orbitals onto a predefined local set of orbitals, which can be e.g. real atomic orbitals or pseudo-atomic orbitals.
The second is to optimize a cost function :math:`f`, which measures the locality of the molecular orbitals.
Because there is no unambiguous choice for the localization criterion, several criteria have been suggested.
Foster-Boys localization minimizes the spread of the orbital
```